test/Test/Toml/Printer/Golden.hs

```diff
@@ -34,17 +35,31 @@ example = mkToml $ do
 
 noFormatting :: PrintOptions
 noFormatting = PrintOptions
-    { shouldSort = False
-    , indent     = 0
+    { printOptionsSorting = Nothing
+    , printOptionsIndent  = 0
     }
 
+-- | Decorate keys as tuples so spam comes before egg
+spamEggDecorate :: Key -> (Int, Key)
+spamEggDecorate k
+    | k == fromText "spam" = (0, fromText "spam")
+    | k == fromText "egg" = (1, fromText "egg")
```

Comment:
You don't need to call `fromText` function here. There is `IsString` instance for `Key`. And we have `OverloadedStrings` language extension enabled project-wide. So you can just write the following and it will work:

```haskell
k == "egg" = (1, "egg")
```

Reply:
Oh that's so much better!